auth/simple_token.go

```diff
@@ -52,6 +54,21 @@ func (as *authStore) assignSimpleTokenToUser(username, token string) {
 	}
 
 	as.simpleTokens[token] = username
+	timer := time.NewTimer(SimpleTokenTTL)
+	donec := make(chan struct{})
+	as.simpleTokenInf[token] = simpleTokenInfo{tokenTimer: timer, done: donec}
+	go func() {
```

Comment:
can we have a single routine that periodically scan the token ttl map and remove the timed out tokens. We do not need per token go routine to keep accuracy.